src/metrics/include/metrics/metrics.hpp

```diff
@@ -86,8 +86,9 @@ using gauged_at_time = std::pair<time_point, gauged_type>;
 
 class Reporter;
 // The v1 namespace is here for two reasons:
-// 1) it's a step towards an ABI. These classes are basically the pimpls of the outer classes
-// 2) it prevents auto-completion of metrics::{X}Impl when you really want metrics::{X}
+// 1) it's a step towards an ABI. These classes are basically the pimpls of the outer classes.
+// 2) it prevents auto-completion of metrics::{X}Impl when you really want metrics::{X}.
+// 3) Counter, Timer and Guage have been deprecated. Use Operation instead when writing actors.
```

Comment:
nit - this isn't one of the reasons why the v1 namespace is here :) Just put a blank line here and nix the number

```c++
// 2) ...
//
// Note: Counter, Timer, and Gauge...
```